qubesadmin/tools/qvm_create.py

```diff
@@ -113,6 +121,14 @@ def main(args=None, app=None):
     if args.one_pool:
         pool = args.one_pool
 
+    if 'disp' in args.properties:
+        args.properties.setdefault('label', 'red')
+        args.cls = 'DispVM'
+        args.properties.setdefault('template', args.properties['disp'])
```

Comment:
`args.disp`. But IMO optional "template" argument to `--disp` may be confusing. Explicit `qvm-create --disp -t ...` is clearer.

Reply:
Is it important _not_ to have the ability to specify a template w/o `-t`?

Reply:
Yes, because it isn't obvious what this argument is. This applies to (almost?) all optional arguments to options. If you add optional command positional arguments to the mix, it is not only confusing to human, but also to the computer - for example in `--opt arg1 arg2`, `arg1` could be interpreted as optional argument to `--opt`, or a positional argument and `--opt1` without its argument.

If you read (in documentation for example) `qvm-create --disp some-name other-name`, you really need to check some extra doc to understand what is going on. On the other hand, `qvm-create --disp --template=some-name other-name` is much clearer.

Reply:
Okay, I get it, will do!